template/deploy/helm/[[product]]-operator/templates/serviceaccount.yaml.j2

```diff
@@ -0,0 +1,75 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+{{- if .Values.serviceAccount.create -}}
```

Comment:
```suggestion
{{ if .Values.serviceAccount.create -}}
```

Otherwise the rendered template looks like this:
```
# This file is automatically generated from the templates in stackabletech/operator-templating
# DON'T MANUALLY EDIT THIS FILE
# =============apiVersion: v1
kind: ServiceAccount
```